explore/src/clue/scala/queries/common/InvitationQueriesGQL.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package queries.common
+
+import clue.GraphQLOperation
+import clue.annotation.GraphQL
+import lucuma.schemas.ObservationDB
+
+object InvitationQueriesGQL:
+  @GraphQL
+  trait CreateInviteMutation extends GraphQLOperation[ObservationDB]:
+    val document: String = s"""
+      mutation inviteToProgram($$programId: ProgramId!, $$recipientEmail: String!) {
```

Comment:
Does the name `inviteToProgram` do anything here. I see we only "name" the mutations here and in the user preferences queries. It doesn't seem as if the name is ever used, though.

Reply:
Not really, I thought it was required to put a name 🤷

Reply:
And I didn't realize it was possible to put a name. 😆